src/games/PlayerVsPlayerGame.js

```diff
@@ -1,16 +1,17 @@
-var $ = require('jquery')
+var $ = require('jquery');
 
 function PlayerVsPlayerGame(ui) {
   this.ui = ui;
+  this.playerMarker = this.ui.displayTurn("X");
 }
 
 PlayerVsPlayerGame.prototype.play = function() {
-  var playerMarker = this.ui.displayTurn("X");
   $(".spot").on("click", (e) => {
+    $(e.target).off('click');
     var id = this.ui.spotClicked(e);
-    this.ui.displayMarker(id, playerMarker);
-    playerMarker = PlayerVsPlayerGame.prototype.switchMarker(playerMarker);
-    this.ui.displayTurn(playerMarker);
+    this.ui.displayMarker(id, this.playerMarker);
+    this.playerMarker = PlayerVsPlayerGame.prototype.switchMarker(this.playerMarker);
+    this.ui.displayTurn(this.playerMarker);
```

Comment:
Behavior in constructor is even weirder since we don't assign the result of this to anything... Kind of a code smell that something is off!